content/efcore/getting-started/samples/manipulating-entities/updating-untracked-entities/Program.cs

```diff
@@ -7,21 +7,21 @@ public static void Main()
     {
         using (var context = new LibraryContext())
         {
-            var untrackedBook = new Book()
-            {
-                Id = 1,
-                Title = "Frankenstein: or, The Modern Prometheus",
-                Genre = "Science Fiction",
-                PublicationYear = 1818
-            };
-
-            context.Books.Update(untrackedBook);
-            context.SaveChanges();
+            var untrackedBook = new Book() 
+			{ 
+				Id = 1, 
+				Title = "To the Lighthouse",
+				Genre = "Literary",
```

Comment:
It doesn't seem to have been committed. The sentence hasn't changed. Also, the corresponding in-tutorial code doesn't adjust `Genre` either.